grammars/fsharp.json

```diff
@@ -928,7 +928,7 @@
                 {
                     "name": "binding.fsharp",
                     "begin": "\\b(let mutable|static let mutable|static let|let inline|let|and|member val|static member inline|static member|default|member|override|let!)(\\s+rec|mutable)?(\\s+\\[\\<.*\\>\\])?\\s*(private|internal|public)?\\s+(\\[[^-=]*\\]|[_[:alpha:]]([_[:alpha:]0-9\\._]+)*|``[_[:alpha:]]([_[:alpha:]0-9\\._`\\s]+|(?<=,)\\s)*)?",
-                    "end": "\\s*(with\\b|=|\\n+=|(?<=\\=))",
+                    "end": "\\s*((with\\b)|(=|\\n+=|(?<=\\=)))",
```

Comment:
This makes it so that the `with` keyword can be captured separately from the `=` symbol. I suspect this regular expression can be simplified, but I did not want to change it further without thinking about it more throughly.